WordPress/UITestsFoundation/Screens/NotificationsScreen.swift

```diff
@@ -102,6 +102,28 @@ public class NotificationsScreen: ScreenObject {
         return self
     }
 
+    public func getNumberOfLikesForNotification() -> (NotificationsScreen, Int) {
+        let totalLikesInString = likeCommentButton.label.prefix(1)
+        let totalLikes = Int(totalLikesInString) ?? 0
+
+        return (self, totalLikes)
+    }
```

Comment:
made this return a tuple with 2 values - the screen (so we can chain the test on iPhone) and the number of likes